mkdocs.yml

```diff
@@ -113,19 +113,13 @@ nav:
     - 'Voting Tickets': 'faq/proof-of-stake/voting-tickets.md'
     - 'Voting Service Providers': 'faq/proof-of-stake/voting-service-providers.md'
 - Advanced:
-  - 'Address Details': advanced/address-details.md
-  - 'Using Testnet': 'advanced/using-testnet.md'
   - 'Manual CLI Installation': 'advanced/manual-cli-install.md'
-  - 'Block Header Specifications': 'advanced/block-header-specifications.md'
```

Comment:
No redirect for this

Reply:
Added redirect, though not sure if working...